bin/flightTracker.js

```diff
@@ -46,10 +49,21 @@ var options = yargs(process.argv.slice(2))
         describe:
           "The airport name as you know. For example John F. Kennedy airport",
         type: "string",
-      });
+      })
+
+      // Will we be passing an access_key to the API?
+
+      // .option("access_key", {
+      //   alias: "k",
+      //   description: "You AviationStack API access key",
+      //   type: "string",
+      //   demandOption: true
+      // })
```

Comment:
This is a good question. The user probably can't run this command until they have fetched the API key, or have manually passed it to CLI. In hindsight I should have changed the `login()` function to `fetchAPIKey()` for when they are using BitWarden.